src/vivarium_public_health/metrics/mortality.py

```diff
@@ -78,40 +81,84 @@ def setup(self, builder: Builder) -> None:
         self.causes_of_death = ["other_causes"] + [
```

Comment:
I believe this is only used if aggregation is false, so you could move it inside the if block